File: src/m/modules/BamgMesher.py
from BamgMesher_python import BamgMesher_python


def BamgMesher(bamgmesh, bamggeom, bamgoptions):
    """BAMGMESHER

    bamgmesh: input bamg mesh
    bamggeom: input bamg geometry for the mesh
    bamgoptions: options for the bamg mesh

    Usage:
        bamgmesh, bamggeom = BamgMesher(bamgmesh, bamggeom, bamgoptions)
    """

    # Call module
    bamgmesh, bamggeom = BamgMesher_python(bamgmesh, bamggeom, bamgoptions)

    return bamgmesh, bamggeom

File: src/m/modules/BamgMesher.m
function [bamgmesh,bamggeom] = BamgMesher(bamgmesh,bamggeom,bamgoptions);
%BAMGMESHER
%
%   Usage:
%      [bamgmesh, bamggeom] = BamgMesher(bamgmesh,bamggeom,bamgoptions);
%
%   bamgmesh: input bamg mesh
%   bamggeom: input bamg geometry for the mesh
%   bamgoptions: options for the bamg mesh

% Check usage
if nargin~=3
	help BamgMesher
	error('Wrong usage (see above)');
end

% Call mex module
[bamgmesh, bamggeom] = BamgMesher_matlab(bamgmesh,bamggeom,bamgoptions);

